docs/en/cookbook/working-with-datetime.rst

```diff
@@ -58,25 +58,88 @@ Handling different Timezones with the DateTime Type
 
 The problem is simple. Not a single database vendor saves the timezone, only the differences to UTC.
 However with frequent daylight saving and political timezone changes you can have a UTC offset that moves
 in different offset directions depending on the real location.
 
-The solution for this dilemma is simple. Don't use timezones with DateTime and Doctrine 2. However there is a workaround
-that even allows correct date-time handling with timezones:
+The solution for this dilemma seems simple. Don't use timezones with DateTime and Doctrine 2. However there are 2 workarounds
+that even allows correct date-time handling with timezones depending on your use case:
 
-1. Always convert any DateTime instance to UTC.
-2. Only set Timezones for displaying purposes
+1a. Don't convert DateTimes to UTC
+1b. Always convert any DateTime instance to UTC
+2. Set Timezones for displaying purposes
 3. Save the Timezone in the Entity for persistence.
 
+Wait: **What? Shall we convert to UTC or not?**
+
+That depends on what kind of DateTimes you are handling! When you are handling current dates like log-entries or 
+datetimes that are within one or two days of the current date, you can - and even should - convert them to UTC. 
+But as soon as you are handling datetimes that are more than a few days in advance (or back) you should **not** convert them to UTC but instead keep them in their local timezone. Why? You might want to read up on `why not to convert a datetime to a timestamp<https://andreas.heigl.org/2016/12/22/why-not-to-convert-a-datetime-to-timestamp/>`_
+
+Handling DateTimes with timezone-informations
+_____________________________________________
+
+Say we have an international calendaring application where users can add events that occur at different places worldwide and therefore at different timezones. To determine the exact time an event will happen means to save both the local time of the event and the timezone the event will happen in.
+
+Using the defalt DateTimeType will store the DateTime in the local time but without the timezone information. Therefore we need to store the timezone information as well and also need to provide a way to get the datetime back from the database with the correct timezone-information.
+
+To be able to transform these values back into their real timezone we have to save the timezone in a separate field of the entity requiring timezoned datetimes:
+
+.. code-block:: php
+
+    <?php
+    namespace Calendaring;
+
+    /**
+     * @Entity
+     */
+    class Event
```

Comment:
Strictly speaking, this is not a valid entity since it has no identifier, can we add one (or make one field one)?

Reply:
I was copy/pasting the example beginning at line 16 here which also does not have an identifier. Shall I add one everywhere? Or shall we skip that here as it is not relevant to the topic illustrated here?

Reply:
I suggest to add them in this file. In some past PRs we also added some missing lines and the identifier doesn't take much space that could distract from the core information. Since there are namespaces in use, some of the annotations need to be included with `use` too

Reply:
This should be addressed now